ui/round/css/_app-layout.scss

```diff
@@ -1,5 +1,8 @@
 .rcontrols {
   grid-area: controls;
+  @include mq-is-col1 {
+    grid-area: moves;
```

Comment:
You want to change the order of the [grid-template-areas property](https://github.com/lichess-org/lila/blob/b10bb1f2e151ee6ea63103dbec0b87f5089cc815/ui/round/css/_app-layout.scss#L51)

Reply:
Oh I see ... it's far more elegant ... let me try

Reply:
Done,

As for the status message: do you think it's better to create the vnode on the fly or always create it and hide/show it via scss ?

Reply:
seems like the additional status message pushes the buttons everyone wants to see further down. maybe it's enough that one can usually tell how the game ended from the board?

Reply:
Consider the same page on a big screen

![Screenshot 2025-02-18 225920](https://github.com/user-attachments/assets/158c18cc-06f4-4311-b792-aa5f7f54af07)

and on a small vertical screen

![Screenshot 2025-02-18 230227](https://github.com/user-attachments/assets/3b9170c2-7250-45c3-9f92-c4dd661db730)

On the big screen the message, the message 'white resigned' is displayed twice: in game meta and in the move table. As none of them is showing in the small vertical screen, I create a new node (was that the question ?)

It's difficult to see that the game has ended, specially in a game without clock.

We could also mimic the app and show a modal with the message.